Physics/Dynamics/FormFactorDecorator.hpp

```diff
@@ -19,62 +14,32 @@ namespace ComPWA {
 namespace Physics {
 namespace Dynamics {
 
-/// The production formfactor is implemented in Blatt-Weisskopf type
-/// \f$B_{L}(q)\f$ Ref. Dalitz Plot Analysis, PDG 2012, where \f[
-/// \frac{B^{\prime}(q_{0})}{B^{\prime}_{L}(q)} = q^{L}B^{\prime}_{L}(q, q_{0})
-/// \f]
-/// \f$B_{L}(q)(i.e., F_L(q))\f$ with \f$L\f$ up to 4 can be found in
-/// BNL-QGS-06-101
-class FormFactorDecorator : public AbstractDynamicalFunction {
+namespace ProductionFormFactor {
```

Comment:
Why is it called ProductionFormFactor? It should be the same for a particle decay.

Reply:
Good question. Yeah I can make it a general FormFactor. I would move all of the code to the FormFactor.hpp and get rid of the FormFactorDecorator then.

Reply:
Moved relevant code to FormFactor.hpp/cpp and removed the decorator ones